package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage && npm run snyk",
+    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage",
```

Comment:
The vulnerabilities are just in the dev dependencies. When I run `snyk wizard` it doesn't give me the option to patch/ignore. I'll update the scripts to just run `snyk test` instead of `snyk test --dev`.